pkgs/tools/system/bashtop/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, bash_5
+, curl
+, lm_sensors
+, procps
+, sysstat
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bashtop";
+  version = "0.8.23";
+
+  src = fetchFromGitHub {
+    owner = "aristocratos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0y6yxm2vmbz0373cfdl6mjh8vhs0r0wcng82n304klx90qxg3ljp";
+  };
+
+  buildInputs = [ bash_5 curl lm_sensors procps sysstat ];
+
+  dontBuild = true;
+  installPhase = ''
+    install -Dm755 ${pname} $out/bin/${pname}
```

Comment:
With [`makeWrapper`](https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper) you can provide the runtime executable dependencies. You dont' have to add them to `buildInputs`, because that only has the effect of setting some environment variables during the build process; not at runtime.